pkg/controller/operandrequest/operandrequest_controller.go

```diff
@@ -259,9 +259,12 @@ func (r *ReconcileOperandRequest) Reconcile(request reconcile.Request) (reconcil
 	}
 
 	// Check if all csv deploy successed
+	if setInstance.Status.Phase == operatorv1alpha1.ClusterPhaseFailed {
+		return reconcile.Result{}, fmt.Errorf("there is one or more operators installation failed")
```

Comment:
Can we get the list of failed operators?

Reply:
@gyliu513 we can get all the operators' status form `Status.Members`, it includes Running, Failed ...., if want to get failed, need some filter for this member list

Reply:
@DanielXLee Yes, I think we need to enhance this, otherwise, I do not know which operators failed from the log message.

Reply:
@gyliu513 I will enhance the status display, @horis233 you can remove the status check part, only fix operator group issue

Reply:
@DanielXLee Ok, I will remove the status check.

Reply:
thanks @DanielXLee @horis233 💯